src/styles/blocks/button-width.scss

```diff
@@ -0,0 +1,23 @@
+.button-width {
+  margin: 0;
+
+  &__style {
```

Comment:
The class name '__style' does not follow BEM naming convention. Consider renaming it to something more descriptive, like '__container', '__content', or a name that relates to the element it is styling.